widgetastic_manageiq/__init__.py

```diff
@@ -2801,10 +2801,15 @@ def __getattr__(self, name):
         return getattr(item, name)
 
     def __str__(self):
-        return str(self._get_existing_entity())
+        return str(self.__repr__())
 
     def __repr__(self):
-        return repr(self._get_existing_entity())
+        try:
+            return repr(self._get_existing_entity())
+        except NoSuchElementException:
+            return '< {c} name {n}, id {id} >'.format(c=self.__class__,
```

Comment:
**Optional**
Consider wrapping `n` and `id` in quotes, so when they're empty strings is obvious there's an empty value there.